file_transfer_protocol/server.py

```diff
@@ -0,0 +1,34 @@
+import socket                   # Import socket module
+
+ONE_CONNECTION_ONLY = True      # Set this to False if you wish to continuously accept connections
+
+filename='mytext.txt'
+port = 12312                    # Reserve a port for your service.
+sock = socket.socket()             # Create a socket object
+host = socket.gethostname()     # Get local machine name
+sock.bind((host, port))            # Bind to the port
+sock.listen(5)                     # Now wait for client connection.
+
+print('Server listening....')
+
+while True:
+    conn, addr = sock.accept()     # Establish connection with client.
+    print('Got connection from', addr)
+    data = conn.recv(1024)
+    print('Server received', repr(data))
+
+    with open(filename,'rb') as out_file:
+        data = out_file.read(1024)
+        while (data):
+           conn.send(data)
+           print('Sent ',repr(data))
+           data = out_file.read(1024)
+
+    print('Done sending')
+    conn.send(b'Thank you for connecting')
```

Comment:
Is this data going to be put into the output file on the other end?

Reply:
no, it should not, I will fix that